src/com/chrisfolger/needsmoredojo/core/amd/importing/ImportReorderer.java

```diff
@@ -111,7 +111,7 @@ public void doSwap(PsiElement source, Editor editor, AMDPsiUtil.Direction direct
 
         int sourceIndex = PsiUtil.getIndexInParent(defines[0]);
         int destinationIndex = PsiUtil.getIndexInParent(defines[1]);
-        JSParameter[] parameterList = items.getFunction().getParameters();
+        JSParameter[] parameterList = items.getFunction().getParameterVariables();
```

Comment:
Also __src/com/chrisfolger/needsmoredojo/core/amd/define/NearestAMDImportLocator.java__ still use the old `getFunction().getParameters()` in line 60 and line 123.

And __src/com/chrisfolger/needsmoredojo/core/amd/importing/ImportUpdater.java__ still use the old `getFunction().getParameters()` in line 65 and line 74.

Reply:
In a lot of cases `getParameters()` and `getParameterVariables()` can be used to do the same things, so it shouldn't really matter which method is used.

Reply:
@TomDevs, thank you fro your reply. You may be right, replacing the minimum possible, but there was a problem reported here: https://github.com/cefolger/needsmoredojo/pull/181#issuecomment-228707162  that was handled by replacing `getFunction().getParameters()` with `getFunction().getParameterVariables()`, as per https://github.com/cefolger/needsmoredojo/pull/181/commits/427bc53d9a210dc17ad06ee0812c121c423dace5. That's why I wanted to point out these places to you to look closer at.